src/main/resources/view/PersonListCard.fxml

```diff
@@ -32,6 +32,7 @@
       <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
       <Label fx:id="prevDateMet" styleClass="cell_small_label" text="\$prevDateMet" />
+      <Label fx:id="salary" styleClass="cell_small_label" text="\$salary" />
```

Comment:
Does this mean salary will be displayed with the name, email, and other labels? If I recall correctly, we would only want to show the salary as a tag